functional_requirements.md

```diff
@@ -0,0 +1,55 @@
+# Haveno Functional Requirements
+
+This document maintains functional requirements for the Haveno user interface (work in progress).
+
+### Startup
+- On first startup, the user must configure a password.
+- On first startup, the user can use a local node or configure a remote node (url, username, password).
+- On startup, the application presents a splash screen with progress and description of startup state.
+
+### Wallet
```

Comment:
Should a user be able to create a new wallet?

Reply:
The application has exactly one "main wallet" which is auto-generated on first startup and doesn't change, so I don't see a use case for generating a new wallet.

Reply:
Alright. Perhaps due to it being a monero wallet, the need is less high to create new wallets. I do know some Monero users doing it to be extra cautious.

Reply:
@deaner0 I think you are referring to subaddress? That should be supported, yes.